pkg/cli/aws/aws_test.go

```diff
@@ -190,28 +190,26 @@ func Test_refreshCredsViaOktaAWSCLI(t *testing.T) {
 }
 
 func Test_oktaAwsCliVersionOutputMatchesV1(t *testing.T) {
```

Comment:
I would've preferred a table test, to be honest. But, that's something we can always change later :D